geode-core/src/main/java/org/apache/geode/internal/process/FileProcessController.java

```diff
@@ -77,6 +77,37 @@
     this.statusTimeoutMillis = units.toMillis(timeout);
   }
 
+  /**
+   * Constructs an instance for controlling a local process.
+   *
+   * @param parameters details about the controllable process
+   *
+   * @throws IllegalArgumentException if pid is not a positive integer
+   */
+  FileProcessController(final FileControllerParameters parameters) {
+    this(parameters, DEFAULT_STATUS_TIMEOUT_MILLIS, MILLISECONDS);
+  }
+
+  /**
+   * Constructs an instance for controlling a local process.
+   *
+   * @param parameters details about the controllable process
+   * @param timeout the timeout that operations must complete within
+   * @param units the units of the timeout
+   *
+   * @throws IllegalArgumentException if pid is not a positive integer
```

Comment:
Delete throws since pid is not a parameter on this constructor.

Reply:
Thanks for pointing this out. Change done!